core/lib/protobuf_config/src/proto/config/eth_sender.proto

```diff
@@ -58,6 +58,7 @@ message Sender {
   reserved 23; reserved "priority_op_start_index";
   optional bool is_verifier_pre_fflonk = 24; // optional
   optional GasLimitMode gas_limit_mode = 25; // optional
+  optional uint64 max_acceptable_base_fee_in_wei = 26; // required; wei
```

Comment:
AFAIU, this parameter has a reasonable default (maybe `u64::MAX` to replicate the previous behavior), so please don't make it required. Generally, introducing required config params is heavily discouraged because it'll break any node with a non-updated config.